keras/utils/training_utils.py

```diff
@@ -171,7 +171,8 @@ def get_slice(data, i, parts):
     # Merge outputs on CPU.
     with tf.device('/cpu:0'):
         merged = []
-        for name, outputs in zip(model.output_names, all_outputs):
+        for n,(name, outputs) in enumerate(zip(model.output_names, all_outputs)):
             merged.append(concatenate(outputs,
-                                      axis=0, name=name))
+                                      axis=0, name=name+'_gpu_'+str(n)))
```

Comment:
Reasonable change, but fix the style:
- `i` for indexing
- use PEP8 syntax (space around operator `+` and after `,`